Script/codecov_downloader.sh

```diff
@@ -2,5 +2,6 @@
 curl -s https://codecov.io/bash >codecov
 VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2)
 for i in 1 256 512; do
-  shasum -a $i -c <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA${i}SUM)
+  shasum -a $i -c --ignore-missing <(curl -s "https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA${i}SUM") ||
+    shasum -a $i -c <(curl -s "https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA${i}SUM" | head -n 1)
```

Comment:
A better way here is to use `grep -w "coverage"` after the pipe (|) to get the line containing shasum for codecov.